index.js

```diff
@@ -34,6 +35,7 @@ io.on('connection', socket => {
 
         socket.on('close', msg => {
             socket.broadcast.emit('message', formatMessage(admin, `${msg.user} ${msg.text}`));
+            socket.emit('redirect', dest);
```

Comment:
lets see if david will approve it

Reply:
looks like a workaround, @Galor-sw please mentioned me next time :)